diff --git a/config.def.h b/config.def.h
index 82b1b09..6065bb6 100644
--- a/config.def.h
+++ b/config.def.h
@@ -84,31 +84,23 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	/* onedark */
+ "#282c34", /* 0: black */
+ "#e06c75", /* 1: red */
+ "#98c379", /* 2: green */
+ "#d19a66", /* 3: yellow */
+ "#61afef", /* 4: blue */
+ "#c678dd", /* 5: magenta */
+ "#56b6c2", /* 6: cyan */
+ "#abb2bf", /* 7: white */
+ "#5c6370", /* 8: brblack */
+ "#e06c75", /* 9: brred */
+ "#98c379", /* 10: brgreen */
+ "#d19a66", /* 11: bryellow */
+ "#61afef", /* 12: brblue */
+ "#c678dd", /* 13: brmagenta */
+ "#56b6c2", /* 14: brcyan */
+ "#ffffff", /* 15: brwhite */
 };
 
 
@@ -116,10 +108,10 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+unsigned int defaultfg = 7;
+unsigned int defaultbg = 0;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
